lib/bumper.js

```diff
@@ -101,15 +84,13 @@ class Bumper {
       return Promise.resolve()
     }
 
-    return this._getOpenPrScope()
-      .then((scope) => {
-        logger.log(`Found a ${scope} bump for the current PR`)
+    return this._getOpenPrInfo()
```

Comment:
Scope is just `patch`/`minor`/`major` info is `scope` + `changelog` I needed to fetch changelog in `check()` as well as `bump()` since I wanted to error in `check()` if the changelog was empty.